### ISSUE TITLE
NPM3D training question

### ISSUE THREAD

**Issue by j2oj2o** [HuguesTHOMAS/KPConv#76]
Hi Thomas

Thank you for making your work publicly available.  I am trying to recreate your results from https://npm3d.fr/paris-lille-3d and do not get the same results (these are validation results).

   mean   |  ground  buildings   poles    bollards trash_cans barriers pedestrians   cars    natural  
----------|------------------------------------------------------------------------------------------
   63.1   |   97.3      89.1      64.1      38.4      27.1      59.7      14.3      89.8      87.9  

I took the information from parameters.txt for the pretrained version but could you please confirm what parameters I should be using, what the validation split should be and if I am correct in using the NPM3D dataset as is i.e. Paris, Lille1 and 2 in the training folder and Dijon etc in the test folder.

Thanks
Justin

**Comment by HuguesTHOMAS** [comment]
Hi @j2oj2o,

The standard validation cloud is Paris, as it is from a different city. If you used the parameters form the pretrained model, they should be pretty good. and you should obtain the same performances as the pretrained model. 

Keep in mind that Paris is a very difficult cloud and the performances at validation are usually worse than on the test set.

**Comment by j2oj2o** [comment]
Thanks, I did use Paris and what you say may explain what I am seeing.
